connector/src/main/java/org/jboss/as/connector/subsystems/datasources/AbstractDataSourceAdd.java

```diff
@@ -62,6 +48,21 @@
 import org.jboss.msc.service.ValueInjectionService;
 import org.jboss.security.SubjectFactory;
 
+import javax.sql.DataSource;
+import java.sql.Driver;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.List;
+
+import static org.jboss.as.connector.subsystems.datasources.Constants.DATASOURCE_DRIVER;
+import static org.jboss.as.connector.subsystems.datasources.Constants.ENABLED;
+import static org.jboss.as.connector.subsystems.datasources.Constants.JNDI_NAME;
+import static org.jboss.as.connector.subsystems.datasources.Constants.JTA;
+import static org.jboss.as.connector.subsystems.datasources.Constants.STATISTICS_ENABLED;
+import static org.jboss.as.connector.subsystems.jca.Constants.DEFAULT_NAME;
+import static org.jboss.as.controller.descriptions.ModelDescriptionConstants.OP_ADDR;
```

Comment:
Please change your IDE to follow the import ordering of the WildFly code base. Thanks.